apps/api/src/city/city.controller.ts

```diff
@@ -12,4 +13,28 @@ export class CityController {
   async getData() {
     return await this.cityService.listCities()
   }
+
+  @Get('list-one/:id')
```

Comment:
Please remove the `list-one/` as it not by the RESTful standard and makes the URL `/city/list-one/12345` instead of `/city/12345`

```suggestion
  @Get(':id')
```